feat: Added complementary fuction `get_offset_holes`.

tuto193 opened the pull request:
Holes can now be retrieved with offset, and other `get/set` functions with offset within GeoFeatures now will use `double` offsets.

kb173 commented:
Thank you! I changed the `get_int_*` to `get_*` and renamed the others to `get_float_*` (and same for the setters) to preserve backwards compatibility. it's great to have both options.

tuto193 commented:
> Thank you! I changed the `get_int_*` to `get_*` and renamed the others to `get_float_*` (and same for the setters) to preserve backwards compatibility. it's great to have both options.

Awesome!
